pkgdb/src/buildenv/assets/mkContainer.nix

```diff
@@ -0,0 +1,26 @@
+# at this point just a thin wrapper around streamLayeredImage, but still worth
+# having to keep as much functionality as possible in lib rather than the module
+# system
+{
+  nixpkgsFlake, # the (bundled) nixpkgs flake,
+  environmentOutPath, # the path to the environment that was built previously
+  system, # the system to build for
+}: let
+  pkgs = nixpkgsFlake.legacyPackages.${system};
+  lowPriority = pkg: pkg.overrideAttrs (old: old // {meta = (old.meta or {}) // {priority = 10000;};});
+
+  buildLayeredImageArgs = {
+    name = "flox-env-container";
+    # symlinkJoin fails when drv contains a symlinked bin directory, so wrap in an additional buildEnv
+    contents = pkgs.buildEnv {
+      name = "contents";
+      paths = [
+        environmentOutPath
+        (lowPriority pkgs.bashInteractive) # for a usable shell
+        (lowPriority pkgs.coreutils) # for just the basic utils
+      ];
+    };
+    config = {};
```

Comment:
nonblocking: make entrypoint configurable by users

Reply:
that requires a change in the env manifest spec.
If we want that it should eb separately ticketed IMO

Reply:
👍